Ports/openrct2/patches/0001-Add-SerenityOS-platform-detection.patch

```diff
@@ -0,0 +1,26 @@
+From 1012a5843aab062c2db59e61c740e429034eb162 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Julian=20Offenh=C3=A4user?= <offenhaeuser@protonmail.com>
+Date: Tue, 7 Feb 2023 21:10:21 +0100
+Subject: [PATCH 1/7] Add SerenityOS platform detection
+
+---
+ src/openrct2/Version.h | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/src/openrct2/Version.h b/src/openrct2/Version.h
+index 0e5d83aaf..428552a4f 100644
```

Comment:
These patches don't look like they have been generated through `./package.sh dev`.

Reply:
Well look who didn't read the documentation :^)